src/config.ts

```diff
@@ -55,6 +56,41 @@ if (envConfigFile) {
   }
 }
 
+// allow to override a few important options by environment variables
+function createServerConfig(host?: string,  port?: string, nameSpace?: string, ssl?: string): ServerConfig {
+  const result = { host, nameSpace } as any;
+
+  if (hasValue(port)) {
+    result.port = Number(port);
+  }
+
+  if (hasValue(ssl)) {
+    result.ssl = ssl.trim().match(/^(true|1|yes)$/i) ? true : false;
+  }
+
+  return result;
+}
+
+const processEnv = {
+  ui: createServerConfig(
+    process.env.DSPACE_HOST,
+    process.env.DSPACE_PORT,
+    process.env.DSPACE_NAMESPACE,
+    process.env.DSPACE_SSL) as ServerConfig,
```

Comment:
These `as ServerConfig` statements are type assertions. Since the `createServerConfig` function now returns a `ServcerConfig` object, they are no longer necessary here. As they don't do any harm either, I won't block the merge for this.

Reply:
If they don't do harm, please merge it as it is. I probably won't find much more time this week.

Reply:
@pnbecker : I removed them on your behalf (since you gave maintainers rights to commit to your branch).  I also tested this change locally, and it seemed to work fine.

So, assuming all works with Travis, I'll merge